openbb_terminal/cryptocurrency/onchain/dune_model.py

```diff
@@ -0,0 +1,134 @@
+import logging
+
+import pandas as pd
+from requests import post
+
+from openbb_terminal.core.session.current_user import get_current_user
+from openbb_terminal.decorators import check_api_key, log_start_end
+from openbb_terminal.helper_funcs import request
+
+logger = logging.getLogger(__name__)
+
+BASE_URL = "https://api.dune.com/api/v1/"
+
+
+def make_api_url(module, action, ID, use_csv=True):
```

Comment:
you default to `use_csv`, but that only works for get-resutls, if you try to `+="/csv"` on execute or status APIs it will fail.

better default to false